### PR TITLE
Minimal ora install

### PR DESCRIPTION
**Pull request by efischer19**
Spinoff of https://github.com/edx/edx-ora2/pull/881, trying to find the minimal amount of files needed to run ORA.

### CODE REVIEW

**Comment by edx-webhook** [comment]
## PR Cover Letter
- [ ] A detailed description of the changes in the body of the PR
- [ ] If available, a link to the relevant JIRA ticket in the body of the PR
  - `[PROJ-XXXX](https://openedx.atlassian.net/browse/PROJ-XXXX)`

## Sandbox
- [ ] Build a sandbox for your branch and add a link here

## Testing
- [ ] i18n
- [ ] RTL
- [ ] Unit, integration, acceptance tests as appropriate
- [ ] Analytics
- [ ] Performance
- [ ] Database migrations are backwards-compatible

## Reviewers
If you've been tagged for review, please check your corresponding box once you've given the :+1:.
- [ ] Code review #1
- [ ] Code review #2
- [ ] Docs review (required for UI strings/error messages)
- [ ] UX review
- [ ] Accessibility review
- [ ] Product review

## Documentation considerations
- [ ] Are there user visible changes? If so, list who is affected, and include a pointer to the relevant code.
- [ ] Should this be mentioned in the release notes? If so, make sure that you included a description.

## DevOps assistance
If it's needed, or even if you suspect it might be needed, do any/all these things as early as possible. Don't surprise devops right before merging! For example, touching envs/aws.py or or envs/common.py are dead giveaways that you'll need assistance when pushing your changes out. Things that require devops attention include, but are not limited to:
- configuration changes
- feature flags
- migrations
Ways to get devops attention, pick-and-choose whichever is most appropriate for your specific PR:
- [ ] Tag devops on your PR (`@edx/devops`), either for review (more specific to devops = better) or just a general FYI
- [ ] File a support ticket

FYI: Tag anyone who might be interested in this PR here.

## Post-review
- [ ] Squash commits

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def load_requirements(*requirements_paths):
     return list(requirements)
 
 setup(
-    name='ora2',
+    name='edx-ora2',
```
Comment (efischer19) [inline setup.py]: This is what gets reported from `pip freeze`, am I correct in thinking we may want the `edx-` prefix there for easier grepping?

**Comment by efischer19** [comment]
First run, no dice. Seems to be looking for un-minified js files.
```
2016-02-25 11:40:16,355 INFO 2625 [openedx.core.djangoapps.common_views.xblock] xblock.py:33 - Failed to load xblock resource
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/common_views/xblock.py", line 29, in xblock_resource
    content = pkg_resources.resource_stream(xblock_class.__module__, uri)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1167, in resource_stream
    self, resource_name
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1725, in get_resource_stream
    return open(self._fn(self.module_path, resource_name), 'rb')
IOError: [Errno 2] No such file or directory: u'/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/openassessment/xblock/static/js/src/oa_shared.js'
2016-02-25 11:40:16,371 INFO 2625 [openedx.core.djangoapps.common_views.xblock] xblock.py:33 - Failed to load xblock resource
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/common_views/xblock.py", line 29, in xblock_resource
    content = pkg_resources.resource_stream(xblock_class.__module__, uri)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1167, in resource_stream
    self, resource_name
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1725, in get_resource_stream
    return open(self._fn(self.module_path, resource_name), 'rb')
IOError: [Errno 2] No such file or directory: u'/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/openassessment/xblock/static/js/src/oa_server.js'
[25/Feb/2016 11:40:16] "GET /static/js/search/course/views/search_form.js HTTP/1.1" 200 231
[25/Feb/2016 11:40:16] "GET /static/js/search/course/views/search_results_view.js HTTP/1.1" 200 1130
[25/Feb/2016 11:40:16] "GET /static/js/bookmarks/views/bookmarks_list_button.js HTTP/1.1" 200 2154
2016-02-25 11:40:16,413 INFO 2625 [openedx.core.djangoapps.common_views.xblock] xblock.py:33 - Failed to load xblock resource
Traceback (most recent call last):
  File "/edx/app/edxapp/edx-platform/openedx/core/djangoapps/common_views/xblock.py", line 29, in xblock_resource
    content = pkg_resources.resource_stream(xblock_class.__module__, uri)
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1167, in resource_stream
    self, resource_name
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1725, in get_resource_stream
    return open(self._fn(self.module_path, resource_name), 'rb')
IOError: [Errno 2] No such file or directory: u'/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/openassessment/xblock/static/js/src/lms'
```

**Comment by efischer19** [comment]
Adding the src js files got things working, but I'm still not able to exclude test files adequately...

File: setup.py [hunk 2]
```diff
@@ -45,7 +45,7 @@ def load_requirements(*requirements_paths):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
     ],
-    packages=find_packages(exclude=["test", "tests"]),
+    packages=find_packages(exclude=['*.test', '*.tests']),
```
Comment (efischer19) [inline setup.py]: From the second answer on http://stackoverflow.com/questions/8556996/setuptools-troubles-excluding-packages-including-data-files, I learned that `setuptools` uses `fnmatchcase`, and was able to interactively run `find_packages()` until I found the proper values for `exclude`.

File: MANIFEST.in [hunk 3]
```diff
@@ -1,8 +1,11 @@
...
 include AUTHORS
 include README.rst
+include openassessment/xblock/static/css/*.css
+include openassessment/xblock/static/js/openassessment*.min.js
+recursive-include openassessment/xblock/static/js/src *.js
+recursive-include openassessment/templates *.html
+recursive-include openassessment/locale *.po
+recursive-include openassessment/locale *.mo
+global-exclude */test*
+global-exclude */test?/*
```
Comment (efischer19) [inline MANIFEST.in]: You may be wondering "Why both of these final two lines?" The answer is that `/` gets treated specially - the first pattern will match *most* of what we want, but numerous instances of `blahblah/test/__init__.py` were still being included. Using both lines gets us where we want.

**Pull request merged**